backend/typescript/services/implementations/creatorService.ts

```diff
@@ -130,6 +162,48 @@ class CreatorService implements ICreatorService {
     }
   }
 
+  async rejectCreator(userId: string): Promise<void> {
+    try {
+      if (!this.emailService) {
+        const errorMessage =
+          "Attempted to call sendCreatorProfileSetupLink but this instance of CreatorService does not have an EmailService instance";
+        Logger.error(errorMessage);
+        throw new Error(errorMessage);
+      }
+
+      const rejectedUser: Creator | null = await Creator.findByPk(
+        Number(userId),
+      );
+
+      if (!rejectedUser) {
+        return;
+      }
+
+      try {
+        const emailBody = `
+        Hello ${rejectedUser.first_name},
```

Comment:
added rejection logic but ticket is blocked by creator data model, getting TS errors because the properties don't exist yet